examples/SLAM-camera/js/index.js

```diff
@@ -62,9 +60,12 @@ function requestSession(mode) {
 function setupButtonsXR() {
     /* Setup AR / VR buttons */
     const arButton = document.getElementById('ar-button');
+   
     if (arButton) {
         arButton.dataset.supported = engine.arSupported;
```

Comment:
You could also do the display by setting this to unsupported. The Button CSS shows the button once supported is set to `true`.